packages/flutter_tools/templates/app_shared/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -1,5 +1,5 @@
 # Project-level configuration.
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Why did this become 3.13?

Reply:
This was required for linking across directories - https://cmake.org/cmake/help/latest/policy/CMP0079.html